sdk/servicebus/service-bus/rollup.base.config.js

```diff
@@ -79,7 +79,8 @@ export function nodeConfig({ test = false, production = false } = {}) {
       "fs",
       "path",
       "@azure/arm-servicebus",
-      "@azure/ms-rest-nodeauth"
+      "@azure/ms-rest-nodeauth",
```

Comment:
In Track 2, we don't have a dependency on `@azure/ms-rest-nodeauth`. So, this should be removed

Reply:
Nice, removed.